src/test/java/org/broadinstitute/hellbender/tools/walkers/MethylationTypeCallerIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package org.broadinstitute.hellbender.tools.walkers;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.File;
+
+public class MethylationTypeCallerIntegrationTest extends CommandLineProgramTest {
+
+    // If true, update the expected outputs in tests that assert an exact match vs. prior output,
+    // instead of actually running the tests. Can be used with "./gradlew test -Dtest.single=MethylationTypeCallerUnitTest"
+    // to update all of the exact-match tests at once. After you do this, you should look at the
+    // diffs in the new expected outputs in git to confirm that they are consistent with expectations.
+    public static final boolean UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS = false;
+
+    public static final String TEST_FILES_DIR = toolsTestDir + "walkers/MethylationTypeCaller/";
```

Comment:
This definition can be removed - you can just call `getToolTestDataDir()`, or `getTestFile("some file")` from within the tests or DataProvider, and the framework will find your test file (as long as the location of the test files matches the location within the class hierarchy of the package of the class being tested - in this case they do since both are in `walkers/MethylationTypeCaller`).